react/ProductDetails.js

```diff
@@ -371,4 +396,7 @@ function mergeSchemaAndDefaultProps(schema, propName) {
   })
 }
 
-export default withRuntimeContext(ProductDetails)
+export default hoistNonReactStatic(
```

Comment:
[withRuntimeContext](https://github.com/vtex-apps/render-runtime/blob/9c1276a99ef7bb43de684f16fd374c08c5d43ff4/react/components/RenderContext.tsx#L14) already does that, no?

Reply:
yeah, but `injectIntl` does not.

Reply:
It does too: https://github.com/yahoo/react-intl/blob/master/src/inject.js

Reply:
nice! didn't know it was released already.